plugins/service-monitor-activemq/pom.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>kylo-service-monitor-activemq</artifactId>
+
+	<parent>
+		<groupId>com.thinkbiganalytics.datalake</groupId>
+		<artifactId>kylo-plugins</artifactId>
+		<version>0.8.3-SNAPSHOT</version>
+	</parent>
+
+	<properties>
+		<plugin.license.parentRelative>../../</plugin.license.parentRelative>
+	</properties>
+
+	<groupId>com.thinkbiganalytics.service.activemq</groupId>
+
+
+	<name>service-monitor-activemq</name>
+
+
+	<dependencies>
+
+		<!-- Define Kylo Dependency -->
+		<dependency>
+			<groupId>com.thinkbiganalytics.datalake</groupId>
+			<artifactId>kylo-service-monitor-core</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+
+		<!-- Define Activemq Dependency -->
+		<dependency>
+			<groupId>org.apache.activemq</groupId>
+			<artifactId>activemq-broker</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.activemq</groupId>
+			<artifactId>activemq-client</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-context-support</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-core</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-annotations</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>3.8.1</version>
```

Comment:
junit dependency management should defer to master pom.xml.  We use version 4.11 currently ...   very soon @jeremymerrifield will change us over to logback  .. so you may want to target that instead.